models/curvefi/ethereum/curvefi_ethereum_sources.yml

```diff
@@ -0,0 +1,30 @@
+version: 2
+
+sources:
+  - name: curvefi_ethereum
+    description: "decoded events and function calls for curve.fi on ethereum"
+    freshness:
+      warn_after: { count: 12, period: hour }
+      error_after: { count: 24, period: hour }
+    tables:
+      - name: CurveFactory_call_deploy_plain_pool
+        loaded_at_field: call_block_time
+      - name: CurveFactory_call_deploy_metapool
+        loaded_at_field: call_block_time
+      - name: CurveFactory_call_deploy_gauge
+        loaded_at_field: call_block_time
+      - name: CurveFactoryV2_evt_CryptoPoolDeployed
+        loaded_at_field: evt_block_time
+      - name: CurveFactoryV2_call_deploy_pool
+        loaded_at_field: call_block_time
+      - name: CurveFactory_evt_LiquidityGaugeDeployed
+        loaded_at_field: evt_block_time
+      - name: CurveFactoryV2_evt_LiquidityGaugeDeployed
+        loaded_at_field: evt_block_time
+  - name: ethereum
```

Comment:
i believe this source would fit into the 'base_sources' model as it's on ethereum schema, not curvefi specific

Reply:
added to "base_sources" model 
I did a different "freshness" as I was thinking contract potentially doesn't get new data as much as txns. But let me know if you feel differently and I will take out the freshness override.

Reply:
yes, custom freshness is good when you know the decoded table refresh rate. we've had recent changes pushed to the repo around editing source freshness dependent on each decoded table